lib/embedded_ansible.rb

```diff
@@ -0,0 +1,24 @@
+class EmbeddedAnsible
+  APPLIANCE_ANSIBLE_DIRECTORY = "/opt/ansible-installer"
```

Comment:
So right now we will be installing the installer (sorry) manually so we will determine this path. When the installer is provided via rpm I assume they will install it wherever they decide so this will likely have to change either way.